metricflow/dataflow/nodes/compute_metrics.py

```diff
@@ -19,7 +20,11 @@ class ComputeMetricsNode(ComputedMetricsOutput):
     """A node that computes metrics from input measures. Dimensions / entities are passed through."""
 
     def __init__(
-        self, parent_node: BaseOutput, metric_specs: Sequence[MetricSpec], for_group_by_source_node: bool = False
+        self,
+        parent_node: BaseOutput,
+        metric_specs: Sequence[MetricSpec],
+        is_aggregated_to_elements: Set[LinkableElementReference],
```

Comment:
nit again: `aggregated_to_elements` - this is no longer a boolean.

Reply:
DANGIT ok I'm going to add that to the top of the stack instead of fixing here